src/ServiceControl/Operations/ErrorQueueImport.cs

```diff
@@ -137,14 +137,16 @@ private void InnerHandle(TransportMessage message)
 
             var failureDetails = failedMessageFactory.ParseFailureDetails(message.Headers);
 
+            var replyToAddress = message.ReplyToAddress == null ? null : message.ReplyToAddress.ToString();
```

Comment:
Given the fact that `ReplyToAddress` is implemented as

```
get
      {
        string destination;
        if (this.Headers.TryGetValue("NServiceBus.ReplyToAddress", out destination))
          return Address.Parse(destination);
        return (Address) null;
      }
```
we should just plain get the header value here and not pass it through the `Address` class. This way we'll preserve whatever the transport-defined value was in the first place.